.github/workflows/update-deployment.yml

```diff
@@ -34,7 +36,7 @@ jobs:
                     --header 'Content-Type: application/json' \
                     --data '{
                         "inputs": {
-                            "instance": "${{ github.event.inputs.instance || \'test\' }}",
+                            "instance": "${{ env.DEPLOY_BRANCH }}",
```

Comment:
Well, this works now, but I'm getting a 404... :thinking:

Reply:
This is working now (calling the dispatch) even though on the other repo that's failing, but it'll be a separate PR in that repo.